modules/project_cleanup/function_source/main.go

```diff
@@ -517,9 +629,15 @@ func invoke(ctx context.Context) {
 	if cleanUpTagKeys {
 		removeTagKeys(organizationId)
 	}
+
+	//Only delete Feeds from deleted projects
+	if cleanUpCaiFeeds {
+		removeFeedsByName(organizationId, IncludedFeeds)
+	}
 }
 
 func CleanUpProjects(ctx context.Context, m PubSubMessage) error {
-	invoke(ctx)
+	IncludedFeeds := getFeedsListFromEnv(TargetIncludedFeeds)
+	invoke(ctx, IncludedFeeds)
```

Comment:
```suggestion
	invoke(ctx)
```
read the variable `TargetIncludedFeeds` the same way the others are read

Reply:
See https://github.com/terraform-google-modules/terraform-google-scheduled-function/blob/master/modules/project_cleanup/function_source/main.go#L59C1-L59C71